documentation/List-of-Action-plugins.md

```diff
@@ -658,6 +659,26 @@ Can supply *either* `applicationId` or `applicationSerial`, although this can be
 
 Note: There is a database trigger/postgres listener to automatically delete files when their database record is deleted, so we only need to delete the records, not the files themselves.
 
+### Aliasing existing template actions
+
+Finally, we have this "special" action which allows us to create an alias to an existing template action. The use for this is if you have configured a complex action with a lot of parameters or queries, and you wish to run it with slight differences for a document preview (for example), you can just set an alias to the original action instead of configuring and maintaining the same thing twice within the template (LINK TO PREVIEW DOCUMENTATION WHEN ITS DONE)
+
+In fact, this is not really an action at all -- it's just a dummy plugin that makes the action library and front-end (and therefore template_actions) see it as an available action. But all the logic is handled internally as part of the `processTrigger` functionality -- if it sees an "alias" action, the function simply swaps it out with the template action it is referencing.
+
+- _Action Code:_ **`alias`**
+
+| Input parameters<br />(\*required) <br/> | Output properties                  |
+| ---------------------------------------- | ---------------------------------- |
+| `code`\*                                 | `output as per the aliased action` |
```

Comment:
hmm, I wonder if there is a short but more explanatory term for this 'codeOfActionToRun` lol

Reply:
The field in `template_action` is called `code` though, so was trying to make it match. I'd be reluctant to change the name of the field, as it's intended to be a unique identifier that could be used for many different purposes -- anything that needs to refer to a specific template_action.

`codeOfActionToRun` doesn't quite work either, as it's not clear you're talking about the *specific* template_action instance as opposed to the "actionCode" which defines the action type. We could probably have been a little clearer overall though, cos there is a little inconsistency in how the term "code" is used elsewhere lol.

Reply:
Sounds good